source/Engine/Bytecode/StandardLibrary.cpp

```diff
@@ -10017,6 +9933,38 @@ VMValue Music_Alter(int argCount, VMValue* args, Uint32 threadID) {
 	AudioManager::AlterMusic(panning, speed, volume);
 	return NULL_VAL;
 }
+/***
+ * Music.GetLoopPoint
+ * \desc Gets the loop point of a music index, if it has one.
+ * \param music (Integer): The music index to get the loop point.
+ * \return Returns the loop point in samples, as an Integer value.
+ * \ns Music
+ */
+VMValue Music_GetLoopPoint(int argCount, VMValue* args, Uint32 threadID) {
+	CHECK_ARGCOUNT(1);
+	ISound* audio = GET_ARG(0, GetMusic);
+	if (!audio) {
+		return NULL_VAL;
+	}
+	return INTEGER_VAL(audio->LoopPoint);
```

Comment:
IMO this should return `null` if `audio->LoopPoint` is `-1` (since there's an absence of a loop point.) It'll also work the same way if `Music.GetLoopPoint(audio)` is passed as an argument to `Music.Play`.